src/translations/qterminal_pt.desktop.yaml

```diff
@@ -1,2 +1,4 @@
+Desktop Entry/Name: "Qterminal"
+Desktop Entry/GenericName: "Emulador de terminal"
```

Comment:
Hmm, authorship by `Sérgio Marques <smarquespt@gmail.com>` is lost and weblate is confused (https://github.com/lxqt/qterminal/commit/dce787632f5558ceb08fe82245dd1b90c8722123) after adding new translation here.

Reply:
I found several commits already merged with the missing quotes.
The doubled ones come from my setting in Featherpad, should change it, some slipped through I see...

Reply:
I'm afraid not if we don't want force pushes to this git repository, so I Ieave a comment here.